pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +244,32 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		Context("Option set to invalid value", func() {
```

Comment:
I'm not too sure of the word invalid.. Don't have a much better proposal though.